synapse/models/inet.py

```diff
@@ -1691,6 +1731,121 @@ def getBaseModels():
                     ('email', {'ptype': 'inet:email', 'ro': 1,
                         'doc': 'The email field parsed from an RFC 2822 address string.'}),
                 )),
+
+                ('inet:http:request', {}, (
+
+                    ('flow', {'ptype': 'inet:flow',
+                        'doc': 'The inet:flow which contained the HTTP request.'}),
+
+                    ('host', {'ptype': 'it:host',
+                        'doc': 'The it:host which sent the HTTP request.'}),
+
+                    ('time', {'ptype': 'time',
+                        'doc': 'The time that the HTTP request was sent.'}),
+
+                    # HTTP protocol specific fields...
+                    ('method', {'ptype': 'str',
+                        'doc': 'The HTTP request method string.'}),
+
+                    ('path', {'ptype': 'str',
+                        'doc': 'The requested HTTP path (without query parameters).'}),
+
+                    ('query', {'ptype': 'str',
+                        'doc': 'The HTTP query string which optionally folows the path.'}),
+
+                    ('body', {'ptype': 'file:bytes',
+                        'doc': 'The body of the HTTP request.'})
+                )),
+
+                ('inet:http:response', {}, (
```

Comment:
Do we want a way to connect requests to responses? Like `inet:flow` but for http?

Reply:
It has a ``request`` property which should be the guid of the originating request.